cmd/main.go

```diff
@@ -0,0 +1,29 @@
+// SPDX-FileCopyrightText: 2023 Comcast Cable Communications Management, LLC
+// SPDX-License-Identifier: LicenseRef-COMCAST
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"runtime/debug"
+
+	"github.com/xmidt-org/caduceus"
+)
+
+func main() {
+	defer func() {
+		if r := recover(); r != nil {
+			fmt.Println("stacktrace from panic: \n" + string(debug.Stack()))
+		}
+	}()
```

Comment:
in terms of logging a panic causing failure, I recommend putting this logic in the fx life cycle 

here's an example of mine: https://github.com/xmidt-org/xmidt-agent/blob/8d8e91fc4e0398c0f0ba864d2cc60ef097772af7/cmd/xmidt-agent/main.go#L203-L268